### ISSUE TITLE
Version dependency in Debian Trixie

### ISSUE THREAD

**Issue by beatquantum** [u-sb/nginx-docs#14]
For experimental purposes, I tried installing nginx-extras on Debian Trixie (Testing) server, for which I used the following source:
https://debian-repo.xxx.sb/nginx/

I get the error below.

root@za01:/etc/apt/sources.list.d# apt install nginx-extras
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

Unsatisfied dependencies:
 lua-resty-core : Depends: libnginx-mod-http-lua (< 1:0.10.27.1~) but 2:1.27.2-2nwtf+340+13trixie1 is to be installed
Error: Unable to correct problems, you have held broken packages.

--------
I checked for any broken packages, but there were none.

root@za01:/etc/apt/sources.list.d# apt install -f
Summary:
  Upgrading: 0, Installing: 0, Removing: 0, Not Upgrading: 0

--------
Of course, Debian Testing is very much expected to break. This is not a high priority at all, but I thought I will share the finding.

**Comment by msg7086** [comment]
Yeah things are constantly changing in Trixie. Will need to take some time to figure it out.

**Comment by msg7086** [comment]
I added a modified lua-resty-core package with relaxed depending versions. It should work to some extent now.
